solidity/deploy/08_transfer_governance.ts

```diff
@@ -0,0 +1,20 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
Should we use this function in deployment scripts now?

Reply:
I'm sorry, I don't get your question. 

We used to transfer the ownership, now we transfer the governance. It's just a function name change, the result is the same.

Reply:
My fault, never mind. 🤦🏻